eos/saveddata/mode.py

```diff
@@ -53,3 +53,8 @@ def calculateModifiedAttributes(self, fit, runTime, forceProjected=False):
             for effect in self.item.effects.itervalues():
                 if effect.runTime == runTime and effect.activeByDefault:
                     effect.handler(fit, self, context=("module",))
+
+    def getValidCharges(self):
+        # Modes don't have charges, but it is queried for so return nothing.
+        validCharges = set()
+        return validCharges
```

Comment:
probably because stances are just an item and often get treated as such. I can generate a stacktrace and find where it's called.